.github/workflows/windows-pr.yml

```diff
@@ -10,7 +10,7 @@ jobs:
         go-version: [1.14.x]
         node-version: [10.x]
         python-version: [3.7]
-        dotnet: ['3.1.100']
+        dotnet: ['3.1.301']
```

Comment:
@stack72 @jaxxstorm any objections against this change?

Reply:
Not from me - if you are happy with the new version being correct then we can roll that out everywhere

Reply:
See my comment above for why I bumped the .NET SDK.

Reply:
@stack72 Which versions are we using "everywhere" now? Could you link to the place where it's configured?

Reply:
https://github.com/dotnet/sourcelink/issues/572#issuecomment-583548551

Reply:
In all of the providers (from a private repo) the version we have configured is 

```
dotnetversion: ['3.1.201'],
```

Reply:
@mikhailshilkov I just did a manual test and I can confirm I could step debug into Pulumi source from my program.

Reply:
![image](https://user-images.githubusercontent.com/750121/86461858-bf3a0100-bd7e-11ea-9320-36f7fca94a76.png)